clusters/build-clusters/01_cluster/machine_config/m5d_machineconfig.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: m5d
```

Comment:
I would name this m5d4x here because different m5d sizes have different amount of disks.

Reply:
Done.